apps/app.js

```diff
@@ -1,19 +1,35 @@
 const express = require("express");
 const app = express();
 
-const { getTopics, getEndpoints } = require("../controllers/controller");
+const {
+  getTopics,
+  getEndpoints,
+  getArticles,
+} = require("../controllers/controller");
 
 app.use(express.json());
 
 app.get("/api/topics", getTopics);
 
 app.get("/api", getEndpoints);
 
+app.get("/api/articles/:id", getArticles);
+
+app.use((err, req, res, next) => {
+  if (err.msg === "Page not found - invalid Id") res.status(404).send(err);
+  next(err);
+});
```

Comment:
we can slightly tweak this if statement to just if (err.msg) as only custom errors have a .msg property. This will keep the custom error handler flexible to any custom error you write with an msg!